<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClothSimDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyGL" name="mygl">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>618</width>
      <height>432</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetClothButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dropCornerButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop Corner</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dropClothButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop</string>
    </property>
   </widget>
   <widget class="QComboBox" name="drawTypeSelection">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Particle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spring</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lambert</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="objTypeSelection">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Cloth</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Box</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FluidSim</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="Spin_W">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>140</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Width">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Height">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>140</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Spin_H">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>140</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label_SpringSize">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>220</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Spring Size</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Spin_SS">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>220</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.200000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="Spin_D">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>180</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Depth">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth</string>
    </property>
   </widget>
   <widget class="QComboBox" name="integrationTypeSelection">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>100</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Verlet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Implicit Euler</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Euler</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1057</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCamera_Controls">
   <property name="text">
    <string>Camera Controls</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGL</class>
   <extends>QOpenGLWidget</extends>
   <header>mygl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
